pygmt/src/text.py

```diff
@@ -24,12 +24,17 @@
     D="offset",
     G="fill",
     N="no_clip",
+    U="timestamp",
     V="verbose",
     W="pen",
     X="xshift",
     Y="yshift",
+    a="aspatial",
     c="panel",
+    e="find",
     f="coltypes",
+    h="header",
+    i="incols",
     p="perspective",
     t="transparency",
```

Comment:
Alias for `w` is needed:
```suggestion
    t="transparency",
    w="wrap",
```